code/game/objects/items/weapons/clown_items.dm

```diff
@@ -39,10 +39,9 @@
 	//So this is a workaround. This also makes more sense from an IC standpoint. ~Carn
 	if(user.client && (target in user.client.screen))
 		to_chat(user, "<span class='notice'>You need to take that [target.name] off before cleaning it.</span>")
-	else if(target == user && user.a_intent == "grab")
+	else if(target == user && user.a_intent == I_GRAB)
 		var/mob/living/carbon/human/muncher = user

```

Comment:
@FlattestGuitar is right. You need to check they are human before this typecast. Otherwise all users will get cast as human here, and then it'll throw a runtime when you try to access the species var as only humans have species